src/main/java/com/mraof/minestuck/tileentity/redstone/WirelessRedstoneTransmitterTileEntity.java

```diff
@@ -127,10 +150,10 @@ public void load(BlockState state, CompoundNBT compound)
 		super.load(state, compound);
 		
 		tickCycle = compound.getInt("tickCycle");
-		int destX = compound.getInt("destX");
-		int destY = compound.getInt("destY");
-		int destZ = compound.getInt("destZ");
-		this.destBlockPos = new BlockPos(destX, destY, destZ);
+		int offsetX = compound.getInt("offsetX");
+		int offsetY = compound.getInt("offsetY");
+		int offsetZ = compound.getInt("offsetZ");
+		this.offsetPos = new BlockPos(offsetX, offsetY, offsetZ);
```

Comment:
Can you include backwards-portability here? It'd involve detecting if you're loading the new format with "offset" coords, or the old format with "dest" coords, and in the case of "dest" coords convert that position to an offset position.